charts/linkerd2/templates/destination.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
     linkerd.io/control-plane-ns: {{.Values.namespace}}
   annotations:
     {{ include "partials.annotations.created-by" . }}
+    config.linkerd.io/opaque-ports: {{.Values.proxy.opaquePorts}}
```

Comment:
For the control plane components, instead of using the user-defined defaults for opaque ports, should we instead use the list of opaque ports we know the control plane components to use?  i.e. we know that the control plane components don't talk to mysql, etc.  I believe this list should just be 443 for talking to the kubernetes API.